modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/resources/rest-openapi.yaml

```diff
@@ -1,24 +1,23 @@
 components:
     schemas:
-        TestEntry:
+        !#objectDefinitionName#!:
             properties:
                 date:
                     description:
-                        The test entry date.
+                        The object date.
                     format: date-time
                     readOnly: true
                     type: string
                     x-field-definition:
-                        name: dateField
-                number:
+                        name: createDate
+                objectField:
                     description:
-                        The test entry number.
-                    format: int64
-                    type: integer
+                        The object custom field.
+                    type: string
                     x-field-definition:
-                        name: numberField
+                        name: !#objectFieldName#!
             x-schema-definition:
-                entityName: "com.liferay.headless.builder.test.model.TestEntry"
+                entityName: "com.liferay.object.model.ObjectDefinition#!#objectDefinitionId#!"
```

Comment:
I am not sure about including the `objectDefinitionId` in the definition, as the objectDefinitionId is internal :thinking: